Polyline/Polyline.swift

```diff
@@ -191,9 +191,9 @@ public func decodePolyline(encodedPolyline: String) -> [CLLocationCoordinate2D]?
 /// :param: encodedPolyline String representing the encoded Polyline
 ///
 /// :returns: A [CLLocation] representing the decoded polyline if valid, nil otherwise
-public func decodePolyline(encodedPolyline: String) -> [CLLocation]? {
+public func decodePolyline(encodedPolyline: String, precision: Double = 1e5) -> [CLLocation]? {
     
-    return decodePolyline(encodedPolyline).map(toLocations)
+    return decodePolyline(encodedPolyline, precision: precision).map(toLocations)
```

Comment:
Throws an error of " Argument labels '(_:,precision:)' do not match any available overloads